lib/companies/schema/company.ex

```diff
@@ -1,27 +1,33 @@
 defmodule Companies.Schema.Company do
   @moduledoc false
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
   alias Companies.Industries
 
-  # @enforce_keys [:industry, :last_changed_on, :location, :name, :website]
+  schema "companies" do
+    field(:blog, :string)
+    field(:description, :string)
+    field(:github, :string)
+    field(:industry, :string)
+    field(:jobs, {:array, :string})
+    field(:last_changed_on, :string)
+    field(:location, :map)
+    field(:name, :string)
+    field(:website, :string)
 
-  defstruct [
-    :blog,
-    :description,
-    :github,
-    :id,
-    :industry,
-    :jobs,
-    :last_changed_on,
-    :location,
-    :name,
-    :website
-  ]
+    timestamps()
+  end
 
   def build(_filename, attrs, _body) do
     struct!(__MODULE__, Map.to_list(attrs))
   end
```

Comment:
Let's rethink this.

Reply:
Yeah, seems weird to have a build function with two of the params not used.